### PR TITLE
Add AS code generation for .proto in massa-proto submodule

### PR DESCRIPTION
**Pull request by bilboquet**

### CODE REVIEW

**Comment by bilboquet** [comment]
@aoudiamoncef && @Leo-Besancon  what's your thoughts?

File: package.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  "main": "index.js",
+  "git repository": "https://github.com/massalabs/massa-proto-as.git",
+  "devDependencies": {
+    "as-proto": "^1.3.0",
+    "as-proto-gen": "^1.3.0",
+    "assemblyscript": "^0.27.5"
+  },
+  "scripts": {
+    "test": "node tests",
+    "asbuild:proto": "protoc --proto_path=massa-proto/proto/abis/ --plugin=protoc-gen-as=./node_modules/.bin/as-proto-gen --as_opt=gen-helper-methods --as_out=./assembly ./massa-proto/proto/abis/massa/abi/v1/abi.proto",
```
Comment (Leo-Besancon) [inline package.json]: For windows, we would need:
```json
"asbuild:proto": "protoc --proto_path=massa-proto/proto/abis/ --plugin=protoc-gen-as=$pwd/node_modules/.bin/as-proto-gen.cmd --as_opt=gen-helper-methods --as_out=./assembly ./massa-proto/proto/abis/massa/abi/v1/abi.proto",
```
(See reference issue here: https://stackoverflow.com/questions/59447763/node-js-grpc-out-protoc-gen-grpc-1-is-not-a-valid-win32-application)
I don't know if npm can discriminate between plateforms. Maybe add that to a `asbuild_win:proto` script and edit the README.md to reflect this?

**Review by Leo-Besancon — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (bilboquet) [reply package.json]: Thanks we solved it!

**Comment by bilboquet** [comment]
Add AS code generation for .proto in massa-proto submodule

**Pull request merged**